Jenkinsfile

```diff
@@ -174,7 +174,7 @@ def buildProcess(String stageKey, String jdkName, String mvnName, goals, options
         try {
             if (makeReports) {
                 jacoco(changeBuildStatus: false,
-                        execPattern: '**/*.exec',
+                        execPattern: '**/target/jacoco*.exec',
```

Comment:
Why you change this line?
It does not make any difference!
Do we have any other tools which produce `exec` files? No, we don't!

Reply:
it's to avoid a deep scanning which have a very poor performance where we know exactly file names we are looking for `jacoco*.exec` and directories. scanning everything is a waste of time.

Reply:
@olamy 
Performance? Open the online logs as me and you will see that the performance is the last problem.